public/app/features/alerting/unified/NotificationPolicies.tsx

```diff
@@ -207,9 +208,8 @@ const AmRoutes = () => {
   if (!selectedAlertmanager) {
     return null;
   }
-  const time_intervals = result?.alertmanager_config ? mergeTimeIntervals(result?.alertmanager_config) : [];
 
-  const numberOfMuteTimings = time_intervals.length;
+  const numberOfMuteTimings = muteTimings?.length || 0;
```

Comment:
I agree - however, the reason that this one is potentially undefined is because it's driven by RTKQ-style hooks, and `muteTimings` is potentially undefined if the API request hasn't yet completed. What this should/could do is show a loading spinner for the count badge while its loading. That could come along with a refactor to refactor the routes so this logic isn't even in here?

I considered returning a loading component if the timings were still fetching, but that would mean the entire policies page would be hidden until a small part of the UI had completed loading